ftw/ftw.yml

```diff
@@ -50,5 +56,5 @@ testoverride:
     '934120-26': 'Rule 934120 partially detected. With HTTP/1.1 Envoy return 400. With HTTP/2 Enclosed alphanumerics not detected. Coraza Side'
     '934120-39': 'Rule 934120 partially detected. With HTTP/1.1 Envoy return 400. With HTTP/2 Enclosed alphanumerics not detected. Coraza Side'
     '932200-13': 'Unfortunate match inside logs against a different rule log. wip'
-    '934131-5': 'See https://github.com/corazawaf/coraza/pull/899'
-    '934131-7': 'See https://github.com/corazawaf/coraza/pull/899'
+    '942440-19': 'Coraza side: Seems like ARGS is not splitted in _GET and _POST in ruleRemoveTargetById. Further investigation needed.'
+    '942440-20': 'Coraza side: Seems like ARGS is not splitted in _GET and _POST in ruleRemoveTargetById. Further investigation needed.'
```

Comment:
The only unexpected behavior that requires further ingestion is related to `942440-19` and `942440-20`. It seems like [the exclusion rules recently added for 942440](https://github.com/coreruleset/coreruleset/blob/40e6687fa2173219cc3c030c6d49088b5e20f375/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf#L1317-L1343) are not working in Coraza. It could be related to the ARGS split into ARGS_GET and ARGS_POST not performed for ARGS that are part of `ctl:ruleRemoveTargetById`